Tasks/DotNetCoreCLIV2/task.json

```
@@ -191,7 +191,7 @@
         "vstsFeed"
       ],
       "type": "pickList",
-      "label": "Use packages from this Azure Artifacts feed",
+      "label": "Use packages from this Azure Artifacts feed. Please just enter the feed name, not an entire url",
```

Our style generally doesn't use "please" in this context. Just tell the user what they need to do.
```suggestion
      "label": "Use packages from this Azure Artifacts feed. Select from the dropdown or enter [project name/]feed name.",
```

---

Changed this!